bgpd/bgp_vty.c

```diff
@@ -5649,9 +5649,11 @@ DEFUN (address_family_encapv6,
 
 DEFUN (address_family_evpn,
        address_family_evpn_cmd,
-       "address-family evpn",
+       "address-family <evpn|l2vpn evpn>",
```

Comment:
As per discussion last week.  We want the syntax to be 'afi safi'.  As such let's make evpn hidden?